README.md

```diff
@@ -58,6 +39,13 @@ Please check out our tutorials:
 Please take a look at [these slides](https://www.slideshare.net/BogdanRoman1/bluebrain-nexus-technical-introduction-91266871)
 for a short technical introduction to Nexus.
 
+# Participation
+
+There are several channels provided to address different issues:
+- **Feature request**: If there is a feature you would like us to implement and it is not listed on the [feature issues](https://github.com/BlueBrain/nexus/issues?q=is%3Aopen+is%3Aissue+label%3Afeature), create a [github issue](https://github.com/BlueBrain/nexus/issues/new?title=[Feature]%20My%20feature%20request) describing your feature with as much detail as possible and tag it with the `feature` label.
+- **Questions**: For any questions on how to deploy/install/use the platform, first please visit the [documentation](https://bluebrain.github.io/nexus/docs/index.html). Alternatively join our [Gitter channel](https://gitter.im/BlueBrain/nexus) to get quick feedback about your inquiry.
```

Comment:
Proposal: "For any questions on how to deploy, install or use the platform, please visit the [documentation](https://bluebrain.github.io/nexus/docs/index.html). If the documentation doesn't contain the answer to your question, you can get in touch with is on our [Gitter channel](https://gitter.im/BlueBrain/nexus)."